app/coffee/ImageOptimiser.coffee

```diff
@@ -12,9 +12,12 @@ module.exports =
 			killSignal: "SIGKILL"
 		exec args, opts,(err, stdout, stderr)->
 			if err?
-				logger.err err:err, stderr:stderr, localPath:localPath, "something went wrong converting compressPng"
+				if err.signal == 'SIGKILL'
```

Comment:
why do nested ifs, what about 

	if err? and err.signal == 'SIGKILL'
		...
	else if err?
		...
	else